interface.go

```diff
@@ -59,6 +59,9 @@ var Err0RTTRejected = errors.New("0-RTT rejected")
 // as well as on the context passed to logging.Tracer.NewConnectionTracer.
 var ConnectionTracingKey = connTracingCtxKey{}
 
+// ConnectionTracingID is the type of the context value saved under the ConnectionTracingKey.
+type ConnectionTracingID uint64
```

Comment:
I'm not really happy with the name. Tracing is an overloaded term, and we already used it for event tracer.

Any suggestions for a better name?

Reply:
I guess this is okay. 
Other ideas that come to my mind are:
```
ConnectionTrackingID
ConnectionSeqID (as the id is sequential)
ConnectionName (used as a name for this connection in tracking)
```

Reply:
Thank you!